src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useEffect, useState } from 'react';
+import { Loader } from '../components/Loader';
+import { PeopleTable } from '../components/PeopleTable';
+import { Person } from '../types';
+import { getPeople } from '../api';
+
+export const PeoplePage: React.FC = React.memo(
+  () => {
+    const [people, setPeople] = useState<Person[]>([]);
+    const [isLoading, setIsLoading] = useState(false);
+    const [isDataLoaded, setIsDataLoaded] = useState(false);
+    const [errorMessage, setErrorMessage] = useState('');
+
+    const loadPeople = async () => {
+      setIsLoading(true);
+
+      try {
+        const loadedPeople = await getPeople();
+
+        const preparedPeople = loadedPeople.map(person => {
+          const mother = loadedPeople.find(mom => (
+            mom.name === person.motherName
+          ));
+          const father = loadedPeople.find(dad => (
+            dad.name === person.fatherName
+          ));
+
+          return (
+            {
+              ...person,
+              mother,
+              father,
```

Comment:
Let's move this function to separate file

Reply:
done